app/src/main/res/values/strings.xml

```diff
@@ -301,13 +301,13 @@
     <string name="advanced_unlock_explanation_summary">Use advanced unlocking to open a database more easily</string>
     <string name="biometric_unlock_enable_title">Biometric unlocking</string>
     <string name="biometric_unlock_enable_summary">Lets you scan your biometric to open the database</string>
-    <string name="biometric_auto_open_prompt_title">Auto-open biometric prompt</string>
-    <string name="biometric_auto_open_prompt_summary">Automatically ask for biometric if the database is set up to use it</string>
+    <string name="biometric_auto_open_prompt_title">Auto-open biometric scan</string>
+    <string name="biometric_auto_open_prompt_summary">Biometric unlocking of all databases set up to use it</string>
```

Comment:
We have already changed this sentence. Not the same meaning here.

Reply:
I keep forgetting. When is the question raised, and what does it do? Will add that as a comment.

Reply:
You already wanted to change it by (pull request reworked 3): "Ask for biometric if the database is set up to require it"
It is a feature that automatically requests the opening of the biometric prompt without pressing the button when a database is already linked to a biometric recognition. It is useful for people who want to go faster. I don't understand why you are making several change proposals if you don't even see what functionality it corresponds to...
https://github.com/Kunzisoft/KeePassDX/wiki/Advanced-Unlocking#d-optional-change-preference-to-quickly-open-the-biometric-prompt

Reply:
```suggestion
    <string name="biometric_auto_open_prompt_summary">Start biometric scan when opening databases using it</string>
```
Require would imply there is no other option, or in the very least that biometric recognition is always part of unlocking.